<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="color-scheme" content="light dark">
  <title>Eric Bonnet</title>
  <!-- Pico.css -->
  <!--<link
  rel="stylesheet"
  href="https://cdn.jsdelivr.net/npm/@picocss/pico@2.1.1/css/pico.min.css"
  >-->
</head>

<body>

<h1><a name="root">Eric Bonnet</a></h1>

<p> <b>W</b>elcome to my home page. I'm a computational biologist working
currently at the french <a
href="https://jacob.cea.fr/drf/ifrancoisjacob/english/Pages/Departments/CNRGH.aspx"
target="_blank">Centre National de Recherche en G&eacute;nomique Humaine</a>
(CNRGH, National Center for Human Genomic Research), located in Evry (France).


The CNRGH is part of the french Commissariat &agrave; l'&Eacute;nergie Atomique
(<a href="https://www.cea.fr/english" target="_blank">CEA, Centre for Atomic
Energy</a>), and is one of the largest sequencing centres in Europe. It is
dedicated to genomic research on human diseases.

I am leading the <a
href="https://jacob.cea.fr/drf/ifrancoisjacob/english/Pages/Departments/CNRGH/Laboratories/bioanalysis.aspx"
target="_blank">computational genomics (bio-analysis) laboratory.</a> We focus on the computational analysis of high-throughput genomic data related to human health and disease.
</p>


<p>Contact: eric-dot-bonnet-at-cnrgh-dot-fr | 
<a href="http://orcid.org/0000-0002-8468-2867" target="_blank">ORCID</a> |
<a href="http://scholar.google.fr/citations?hl=fr&user=IE-sDWsAAAAJ&oi=sra" target="_blank">Google Scholar</a> |
<a href="https://www.linkedin.com/in/ericbonnet7" target="_blank">LinkedIn</a> |
<a href="https://github.com/erbon7/" target="_blank">GitHub</a>  
</p>

<p>
<ul>
<li><a href="#exp">Professional experience, education and training</a></li>
<li><a href="#pub">Publications</a></li>
<li><a href="#talks">Talks</a></li>
<li><a href="#soft">Software and models</a></li>
<li><a href="#grants">Grants and projects, teaching, referee, supervision, workshops organization and PhD committees</a></li>
</ul>
</p>

<h2><a name="exp"></a>Professional experience</h2>

<p>
<a href="https://www.cea.fr/english" target="_blank"><img src="png/cea.png" height=60></a>
<a href="https://jacob.cea.fr/drf/ifrancoisjacob/english/Pages/Departments/CNRGH.aspx" target="_blank"><img src="png/CNRGH_vignette_up.png" height=60></a>
<a href="https://science.institut-curie.org/" target="_blank"><img src="png/logo_fr.png" height=60 ></a>
<a href="https://vib.be/en#/" target="_blank"><img src="png/vib.png" height=60></a>
<a href="https://www.vandepeerlab.be/" target="_blank"><img src="png/logo_bsb_75_hoog.png" height=60></a>
</p>

<p>
<ul>
<li><b>[ 2016 - now ]</b>  CEA research director and team leader at
the french <a href="https://www.cea.fr/english" target="_blank"> Commissariat &agrave;
l'&Eacute;nergie Atomique</a> (CEA, Center for Atomic Energy), I'm working at the Centre National
de Recherche en G&eacute;nomique Humaine (CNRGH, National Center for Human Genomic Research) in Evry (France). Team leader for the <a
href="https://jacob.cea.fr/drf/ifrancoisjacob/english/Pages/Departments/CNRGH/Laboratories/bioanalysis.aspx" target="_blank"> Computational Genomics (Bio-analysis) Laboratory
</a>.</li> 

<li><b>[ 2010 - 2015 ]</b>  Research Engineer at the <a
href="http://www.curie.fr" target="_blank">Institut Curie</a> (Paris), in the
<a href="http://bioinfo-out.curie.fr/sysbio/"> Team Computational Systems
Biology of Cancer</a>.</li>

<li><b>[ 2001 - 2010 ]</b> Post-doc at the VIB Department of Plant Systems
Biology, Gent University (Belgium), in the <a
href="https://www.vandepeerlab.be/" target="_blank">Team Bioinformatics
and Evolutionary Genomics</a>.  </li>

<li><b>[ 1997 - 2001 ]</b> Junior consultant and developer for computer science
companies (SSII) in Paris, France. </li> 

<li><b>[ 1997 ]</b> Post-doc (6 months) at the Laboratoire d'entomologie, Mus&eacute;um National d'Histoire Naturelle (MNHN, Paris, France) with <a href="pdf/Metaleptea_30_3.pdf" target="_blank">Dr Christiane Am&eacute;d&eacute;gnato &dagger;.</a> </li> </ul>
</p>

<h2><a name="edu"></a>Education</h2>

<p>
<ul>
<li><b>[ 2021 ]</b> HDR (Habilitation &agrave; Diriger les Recherches). University of Evry Val d'Essone (UEVE, Paris-Saclay).</li>
<li><b>[ 1994 - 1996 ]</b> PhD in computational ecology "Phylogeny and
resource sharing in <i>Ortoptera Acrididae</i>". University of Limoges (France), supervisor
prof. <a href="https://www.researchgate.net/profile/Daniel_Petit2" target="_blank">Daniel Petit</a> (summa cum laude).</li>
<li><b>[ 1993 - 1994 ]</b> DEA Neurosciences, University of Toulouse (France).</li> 
<li><b>[ 1989 - 1993 ]</b> Master's degree in Biology, University of Poitiers (France).</li> 
</ul>
</p>


<h2><a name="pub"></a>Publications</h2>
<p> (*) indicates co-authorship</p>

<p>K Muret, V Le Goff, C Dandine-Roulland, C Hotz, F Jean-Louis, B Boisson, L
Mesrob, F Sandron, D Daian, R Olaso, E Le Floch, V Meyer, P Wolkenstein,
JLCasanova, Y L&eacute;vy, E Bonnet, JF Deleuze* and S Hue*. (2024).
<b>Comprehensive Catalog of Variants Potentially Associated with Hidradenitis
Suppurativa, Including Newly Identified Variants from a Cohort of 100
Patients</b>, <i> Int. J. Mol. Sci. </i>,  2024, 25, 10374.  <a
href="https://doi.org/10.3390/ijms251910374" target="_blank">DOI</a></p>

<p>D Derbala, A Garnier, E Bonnet, JF Deleuze, J Tost (2024) <b>Whole-Genome
Bisulfite Sequencing Protocol for the Analysis of Genome-Wide DNA Methylation
and Hydroxymethylation Patterns at Single-Nucleotide Resolution</b>. Epigenome
Editing: Methods and Protocols, 353-382. <a
href="https://doi.org/10.1007/978-1-0716-4051-7_18" target="_blank">DOI</a>

<p>Bonnet E. (2023). <b> La transcriptomique en cellule unique pour
&eacute;tudier des maladies neurod&eacute;g&eacute;n&eacute;ratives.</b>
<i>L'Actualit&eacute; Chimique</i>, num&eacute;ro 490, D&eacute;cembre
2023.  <a href="pdf/eb_actu_chim_2023.pdf">pdf</a> <a href="https://cea.hal.science/cea-04394836" target="_blank">HAL preprint</a>    </p> 

<p>Derbois C, Palomares MA, Deleuze JF, Cabannes E, Bonnet E. (2023) <b> Single
cell transcriptome sequencing of stimulated and frozen human peripheral blood
mononuclear cells. </b> <i>Sci Data</i>, 2023 Jul 6;10(1):433, <a href="
https://doi.org/10.1038/s41597-023-02348-z" target="_blank">DOI</a></p>

<p>M Riquelme-Perez, F Perez-Sanz, JF Deleuze, C Escartin, E Bonnet*, S
Brohard*. (2023) DEVEA: an interactive shiny application for Differential Expression
analysis, data Visualization and Enrichment Analysis of transcriptomics data.
<i>F1000Research</i>, <a href="https://f1000research.com/articles/11-711"
target="_blank">DOI</a>.</p>

<p>Julien Guemri , Morgane Pierre-Jean, Sol&egrave;ne Brohard, Nouara
Oussada, Caroline Horgues, Eric Bonnet, Florence Mauger and
Jean-Fran&ccedil;ois Deleuze. (2022) <b>Methylated ccfDNA from plasma biomarkers
of Alzheimer's disease using targeted bisulfite sequencing</b>. <i>Epigenomics</i>, vol
14, no 8, <a href="https://doi.org/10.2217/epi-2021-0491"
target="_blank">DOI</a>.</p>

<p>L. Abjean,  L. Ben Haim,  M.  Riquelme-Perez, P. Gipchtein, C.
Derbois, M.A.  Palomares, F. Petit,  A.S. H&eacute;rard, M.C. Gaillard, M.
Guillermier, M.  Gaudin-Gu&eacute;rif, N. Sagar, N. Dufour, N. Robil,  M. Kabani, R.
Melki,  P. De la Grange, A.P. Bemelmans, G. Bonvento, J.F.  Deleuze, P.
Hantraye,  E. Bonnet, S.  Brohard, R. Olaso, E. Brouillet,  M.A. Carrillo-de
Sauvage, C. Escartin. (2022) <b>The JAK2-STAT3 pathway controls a beneficial
proteostasis response of reactive astrocytes in Huntington's disease</b>. <i>Brain</i>, <a href="https://doi.org/10.1093/brain/awac068" target="_blank">DOI</a>.</p>

<p>Eric Bonnet. (2021) <b>Using convolutional neural networks for the classification of breast cancer images</b>. arXiv  <a href="https://arxiv.org/abs/2108.13661"
target="_blank">DOI</a>. </p>

<p>Vidisha Singh, George D.Kalliolias, Marek Ostaszewski, Ma&euml;va
Veyssiere, Eleftherios Pilalis, Piotr Gawron, Alexander Mazein, Eric Bonnet,
Elisabeth Petit-Teixeira and Anna Niarakis. (2020) <b>RA-map: building a
state-of-the-art interactive knowledge base for rheumatoid arthritis</b>.
<i>Database</i>, Volume 2020, 2020, baaa017, <a href="https://DOI.org/10.1093/database/baaa017" target="_blank">DOI</a>
</p>

<p>Marie-Ange Palomares*, Cyril Dalmasso*, Eric Bonnet*, C&eacute;line
Derbois, Sol&egrave;ne Brohard-Julien, Christophe Ambroise, Christophe Battail,
Jean-Fran&ccedil;ois Deleuze, Robert Olaso. (2019) <b>Systematic analysis of TruSeq,
SMARTer and SMARTer Ultra-Low RNA-seq kits for standard, low and ultra-low
quantity samples</b>. <i>Scientific Reports</i> 9 7550 <a href="https://DOI.org/10.1038/s41598-019-43983-0" target="_blank">DOI</a>.</p>

<p>Eric Bonnet*, Marie-Laure Moutet*, C&eacute;line Baulard*, Delphine
Bacq-Daian, Florian Sandron, Lilia Mesrob, Bertrand Fin, Marc Del&eacute;pine,
Marie-Ange Palomares, Claire Jubin, H&eacute;l&egrave;ne Blanch&eacute;,
Vincent Meyer, Anne Boland, Robert Olaso, Jean-Fran&ccedil;ois Deleuze. (2018)
<b>Performance comparison of three DNA extraction kits on human whole-exome
data from formalin-fixed paraffin-embedded normal and tumor samples.</b>
<i>PLOS ONE</i>, <a href="https://DOI.org/10.1371/journal.pone.0195471"
target="_blank">DOI</a></p>

<p>Pau Erola, Eric Bonnet, Tom Michoel. <b>Learning differential module
networks across multiple experimental conditions.</b> (2018) book chapter, <i>
Gene regulatory network inference</i>, Humana Press. <a href="https://www.springer.com/la/book/9781493988815" target="_blank">Book</a> <a
href="https://arxiv.org/abs/1711.08927" target="_blank">arXiv preprint</a></p> 

<p>Kernaleguen M., Daviaud C., Shen Y., Bonnet E., Renault V., Deleuze J.F.,
Mauger F. and Tost J. <b>Whole genome bisulfite sequencing for the analysis
of genome-wide DNA methylation and hydroxymethylation patterns at
single-nucleotide resolution.</b> (2018)  <i>Methods Mol Biol.</i> 1767:311-349. DOI: 10.1007/978-1-4939-7774-1_18.</p>

<p>Fatemeh Seyednasrollah, Devin C. Koestler, Tao Wang, Stephen R. Piccolo,
Roberto Vega, Russell Greiner, Christiane Fuchs, Eyal Gofer, Luke Kumar,
Russell D. Wolfinger, Kimberly Kanigel Winner, Chris Bare, Elias Chaibub Neto,
Thomas Yu, Liji Shen, Kald Abdallah, Thea Norman, Gustavo Stolovitzky, Howard
R. Soule, Christopher J. Sweeney, Charles J. Ryan, Howard I. Scher, Oliver
Sartor, Laura L. Elo, Fang Liz Zhou, Justin Guinney, James C. Costello, and
Prostate Cancer DREAM Challenge Community. <b> A DREAM Challenge to Build
Prediction Models for Short-Term Discontinuation of Docetaxel in Metastatic
Castration-Resistant Prostate Cancer.</b>(2017) <i>JCO Clinical Cancer
Informatics</i> <a href="http://ascopubs.org/DOI/10.1200/CCI.17.00018"
target="_blank">web</a> (as contributing team).</p>


<p>Arnau Montagud, Pauline Traynard, Loredana Martignetti, Eric Bonnet,
Emmanuel Barillot, Andrei Zinovyev, Laurence Calzone. <b>Conceptual and
computational framework for logical modelling of biological networks
deregulated in diseases. </b> (2017) <i>Briefings in Bioinformatics</i>,
bbx163. <a href="https://DOI.org/10.1093/bib/bbx163" target="_blank">DOI</a>
<a href="https://academic.oup.com/bib/advance-article-abstract/DOI/10.1093/bib/bbx163/4716963"
target="_blank">web</a></p>

<p>Paul Deveau, Emmanuel Barillot, Valentina Boeva, Andrei Zinovyev, Eric
Bonnet. <b>Calculating biological module enrichment or depletion and visualizing
data on large-scale molecular maps with ACSNMineR and RNaviCell R packages. </b>
(2016) <i>The R Journal</i>. The R Journal Vol. 8/2, December 2016, ISSN 2073-4859.
<a href="https://journal.r-project.org/archive/2016/RJ-2016-047/RJ-2016-047.pdf" target="_blank">pdf</a>
<a href="http://biorxiv.org/content/early/2016/07/22/064469" target="_blank">bioRxiv</a> </p>


<p>Loredana Martignetti, Laurence Calzone, Eric Bonnet, Emmanuel Barillot,
Andrei Zinovyev. <b>ROMA: representation and quantification of module activity
from target expression data.</b> <i>Frontiers in genetics</i> 7, 18. <a href="https://doi.org/10.3389/fgene.2016.00018" target="_blank">DOI</a></p>

<p>Inna Kuperstein, Eric Bonnet, Hien-Anh Nguyen, David Cohen, Eric
Viara, Luca Grieco, Simon Fourquet, Laurence Calzone, Christophe Russo, Maria
Kondratova, Marie Dutreix, Emmanuel Barillot and Andrei Zinovyev. 
<b>Atlas of Cancer Signaling Network: a systems biology resource for
integrative analysis of cancer data with Google Maps.</b> (2015) <i>Oncogenesis</i>.
2015 Jul 20;4:e160. DOI: 10.1038/oncsis.2015.19. 
<a href="http://www.nature.com/oncsis/journal/v4/n7/full/oncsis201519a.html" target="_blank">web</a> </p> 

<p>Eric Bonnet, Eric Viara, Inna Kuperstein, Laurence Calzone, David PA
Cohen, Emmanuel Barillot, Andrei Zinovyev. <b>NaviCell Web Service for
Network-based Data Visualization.</b> (2015) <i>Nucleic Acids Research</i> Web issue,
DOI: 10.1093/nar/gkv450. <a href ="http://nar.oxfordjournals.org/content/early/2015/05/09/nar.gkv450.full" target="_blank">web</a>
<a href="http://nar.oxfordjournals.org/content/early/2015/05/09/nar.gkv450.full.pdf+html" target="_blank">pdf</a></p>

<p>Eric Bonnet, Laurence Calzone, Tom Michoel. <b>Integrative
multi-omics module network inference with Lemon-Tree.</b> (2015) <i>PLoS Comput
Biol</i> 11(2): e1003983. DOI:10.1371/journal.pcbi.1003983 <a
href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003983" target="_blank">web</a>
<a href="http://arxiv.org/abs/1408.0472" target="_blank">arXiv</a></p> 

<p>Laurence Calzone, Inna Kuperstein, David Cohen, Luca Grieco, Eric
Bonnet, Nicolas Servant, Philippe Hup&eacute;, Andrei Zinovyev, Emmanuel
Barillot. <b>Mod&eacute;lisation de r&eacute;seaux biologiques et m&eacute;decine de pr&eacute;cision en
oncologie.</b> (2014) <i>Bulletin du Cancer</i> Vol. 101, Issue 1, pp 18-21. <a href="pdf/26.pdf">pdf</a></p>

<p>Dagmar Waltemath, Frank T Bergmann, Claudine Chaouiya, Tobias Czauderna,
Padraig Gleeson, Carole Goble, Martin Golebiewski, Mickael Hucka, Nick Juty,
Olga Krebs, Nicolas Le Nov&egrave;re, Huaiyu Mi, Ion I Moraru, Chris J Myers,
David Nickerson, Brett G Olivier, Nicolas Rodriguez, Falk Schreiber, Lucian
Smith, Fengkai Zhang, Eric Bonnet.  <b> Meeting report from the fourth meeting
of the Computational Modeling in Biology Network (COMBINE).  </b> (2014)
<i>Standards in Genomic Sciences.</i> Vol 9, No 3. <a
href="http://www.standardsingenomics.org/index.php/sigen/article/view/527/1135"
target="_blank">web</a>.  </p>

<p>Margolin AA, et al.  <b>Systematic analysis of challenge-driven
improvements in molecular prognostic models for breast cancer.</b> (2013)
<i>Sci Transl Med.</i> Apr 17;5(181):181re1. <a
href="http://stm.sciencemag.org/content/5/181/181re1.short"
target="_blank">web</a> (as a contributing team with JP Vert).</p>

<p>Vera-Licona, P., Bonnet, E., Barillot, E., Zinovyev,
A. (2013) <b>OCSANA: Optimal Combinations of Intervention Strategies for Network
Analysis.</b> <i>Bioinformatics.</i> 29 (12), 1571-1573. <a
href="http://bioinformatics.oxfordjournals.org/content/29/12/1571" target="_blank">
web</a>
<a href="pdf/24.pdf">pdf</a>
</p>

<p>Bonnet, E., Calzone, L., Rovera, D., Stoll, G., Barillot, E., Zinovyev, A.
(2013) <b>BiNoM, a Cytoscape plugin for accessing and analyzing pathways using
standard systems biology formats</b>, <i>BMC Systems Biology</i>, 7:18.
<a href="http://www.biomedcentral.com/1752-0509/7/18" target="_blank">web</a>
</p>

<p>Bonnet, E., Calzone, L., Rovera, D., Stoll, G., Barillot, E., Zinovyev, A.
(2013) <b>Practical use of BiNoM, a Biological Network Manager software.</b> In
<i>In silico Systems Biology: A systems-based approach to understanding
biological processes </i>, Humana Press, pp 127-146. <a href="pdf/22.pdf">pdf</a></p>


<p>The Ectocarpus genome consortium. <b>The Ectocarpus genome and
brown algal genomics</b>. (2012) In <i>Genomic insights gained into the Diversity,
Biology and Evolution of Algae</i>, Advances in Botanical Research, Vol. 64, pp 143-174, Elsevier, ISSN 0065-2296.
<a href="http://store.elsevier.com/Genomic-Insights-into-the-Biology-of-Algae/isbn-9780123914996/" target="_blank">web</a>
</p>

<p>Grbi&#263;, M., Thomas Van Leeuwen, T., Clark, R.M., Rombauts, S., Rouz&eacute;, P.,
Grbi&#263;, V., Osborne, E.J., Dermauw, W., Phuong Cao, T.N., Ortego, F.,
Hern&aacute;ndez-Crespo, P., Diaz, I., Martinez, M., Navajas, M., Sucena, E.,
Magalh&atilde;es, S., Nagy, L., Pace, R.M., Djuranovi&#263;, S., Smagghe, G., Iga, M.,
Christiaens, O., Veenstra, J.A., Ewer, J., Mancilla Villalobos, R., Hutter,
J.L., Hudson, S.D., Velez, M., Yi, S.Y., Zeng, J., Pires-daSilva, A., Roch, F.,
Cazaux, M., Navarro, M., Zhurov, V., Acevedo, G., Bjelica, A., Fawcett, J.A.,
Bonnet, E., Martens, C., Baele, G., Wissler, L., Sanchez-Rodriguez, A.,
Tirry, L., Blais, C., Demeestere, K., Henz, S.R., Gregory, T.R., Mathieu, J.,
Verdon, L., Farinelli, L., Schmutz, S., Lindquist, E., Feyereisen, R., Van de
Peer, Y. <b>The genome of Tetranychus urticae reveals herbivorous pest
adaptations</b>. (2011) <i>Nature</i> 479, 487-492 
DOI:10.1038/nature10640  
<a href="pdf/18.pdf">pdf</a>
<a href="http://www.nature.com/nature/journal/v479/n7374/full/nature10640.html" target="_blank">web</a> 
<a href="pdf/spider_mite_sup_info.pdf" >supplementary information</a>
</p>

<p>Bonnet, E. Integrating microRNA expression data and other
sources in module network reconstruction</b>. (2010) In <i>Proceedings of the 6th workshop on
computation of biochemical pathways and genetic networks</i>, Logos Verlag Berlin
GmbH. <a href="pdf/17.pdf">pdf</a></p>
 
<p>Bonnet, E., Michoel, T. and Van de Peer, Y. <b>Prediction of a gene
regulatory network linked to prostate cancer from gene expression, microRNA and
clinical data</b>. (2010) <i>Bioinformatics</i>, 26 (18): i638-i644. <a href="pdf/16.pdf">pdf</a>
 <a href="http://bioinformatics.oxfordjournals.org/content/26/18/i638.full" target="_blank">web</a>
</p>

<p>Michoel, T., Joshi, A., Bonnet, E., Vermeirssen, V. and Van de Peer, Y.
<b>Towards system level modeling of functionnal modules and regulatory
pathways using genome-scale data</b>. (2010) <i>Proceedings of the Seventh International
Workshop on Computational Systems Biology</i>, WCSB.</p> 

<p>Bonnet, E., Tatari, M., Joshi, A., Michoel, T., Marchal, K., Beerx, G.,
Van de Peer, Y. <b>Module network inference from cancer expression dataset
identifies microRNAs as high scoring regulators</b>. (2010) <i>PLoS ONE</i> 5(4):
e10162.DOI:10.1371/journal.pone.0010162. <a href="pdf/2010_plos_one.pdf">pdf</a>
<a href="http://www.plosone.org/article/info:DOI%2F10.1371%2Fjournal.pone.0010162" target="_blank">web</a>
</p>

<p>Bonnet, E., He, Y., Billiau, K., Van de Peer, Y. <b>TAPIR, a web
server for the prediction of plant microRNA targets, including target mimics</b>.
 (2010) <i>Bioinformatics,</i> 26(12), 1566-1568. <a href="pdf/tapir.pdf">pdf</a>
<a href="http://bioinformatics.oxfordjournals.org/content/26/12/1566.abstract" target="_blank">web</a>
</p>

<p>Cock, M., Sterck, L., Rouz&eacute;, P., Scornet, D., Allen, A., Amoutzias, G.,
Anthouard, V., Artiguenave, F., Aury, J., Badger, J., Beszteri, B., Billiau,
K., Bonnet, E., Bothwell, J., Bowler, C., Boyen, C., Brownlee, C., Carrano, C.,
Charrier, B., Youn Cho, G., Coelho, A., Colln, J., Corre, E., Da Silva, C.,
Delage, L., Delaroque, N., Dittami, S., Doulbeau, S., Elias, M., Farnham, G.,
Gachon, C., Gschloessl, B., Heesch, S., Jabbari, K., Jubin, C., Kawai, H.,
Kimura, K., Kloareg, B., Kupper, F., Lang, D., Le Bail, A., Leblanc, C.,
Lerouge, P., Lohr, M., Lopez, P., Martens, C., Maumus, F., Michel, G.,
Miranda-Saavedra, D., Morales, J., Moreau, H., Motomura, T., Nagasato, C.,
Napoli, C., Nelson, D., Nyvall-Coll&eacute;n, P., Peters, S., Pommier, C., Potin, P.,
Poulain, J., Quesneville, H., Read, B., Rensing, S.A., Ritter, A., Rousvoal,
S., Samanta, M., Samson, G., Schroeder, D., Sgurens, B., Strittmatter, M.,
Tonon, T., Tregear, J., Valentin, K., von Dassow, P., Yamagishi, T., Van de
Peer, Y., Wincker, P. <b>The Ectocarpus genome and the independent
evolution of multicellularity in the brown algae</b>. (2010) <i>Nature</i>, 465, 617-621. <a href="pdf/12.pdf">pdf</a>
<a href="http://www.nature.com/nature/journal/v465/n7298/full/nature09016.html" target="_blank">web</a> 
<a href="pdf/ecto_sup_info.pdf" >supplementary information</a>
</p>

<p>Vermeirssen, V., Joshi, A., Michoel, T., Bonnet, E., Casneuf, T. and Van
de Peer, Y. <b>Transcription regulatory networks in Caenorhabditis elegans
inferred through reverse-engineering of gene expression profiles constitute
biological hypotheses for metazoan development</b>. (2009) <i>Mol. BioSyst.,</i> 5, 1817-1830. <a href="pdf/11.pdf">pdf</a>
<a href="http://pubs.rsc.org/en/Content/ArticleLanding/2009/MB/b908108a" target="_blank">web</a>
</p>

<p>Abeel, T., Saeys, Y., Bonnet, E., Rouz&eacute;, P., Van de Peer, Y. 
<b>Generic eukaryotic core promoter prediction using structural features of DNA</b>.
 (2008) <i>Genome Research</i> 18, 310-23. <a href="pdf/10.pdf">pdf</a>
<a href="http://genome.cshlp.org/content/18/2/310.abstract" target="_blank">web</a> 
</p>

<p>Van Leene, J., Stals, H., Eeckhout, D., Persiau, G., Van De Slijke, E., Van
Isterdael, G., De Clercq, R., Bonnet, E., Laukens, K., Remmerie, N., Hendrickx,
K., De Vijlder, T., Abdelkrim, A., Pharazyn, A., Van Onckelen, H., Witters, E.,
Inz&eacute;, D., De Jaeger, G. <b>A Tandem Affinity Purification-based technology
platform to study the cell cycle interactome in Arabidopsis thaliana</b>. (2007) <i>Mol. Cel.
Prot.</i> 6(7):1226-38. <a href="pdf/9.pdf">pdf</a>
<a href="http://www.mcponline.org/content/6/7/1226.full" target="_blank">web</a>
</p>

<p>Michoel, T., Maere, S., Bonnet, E., Joshi, A., Saeys, Y., Van den Bulcke,
T., Van Leemput, K., van Remortel, P., Kuiper, M., Marchal, K., Van de Peer, Y.
<b>Validating module networks learning algorithms using simulated data</b>. BMC
 (2007) <i>Bioinformatics</i> 8, S5. <a href="pdf/8.pdf">pdf</a>
<a href="http://www.biomedcentral.com/1471-2105/8/S2/S5" target="_blank">web</a></p>

<p>Bonnet, E., Van de Peer, Y., Rouz&eacute;, P. <b>The small RNA world of
plants</b>. (2006) <i>New Phytol.</i> 171, 451-68. (Tansley Review). 
<a href="http://onlinelibrary.wiley.com/DOI/10.1111/j.1469-8137.2006.01806.x/full" target="_blank">web</a>
<a href="pdf/7.pdf">pdf</a></p>

<p>Faes, P., Minnaert, B., Christiaens, M., Bonnet, E., Saeys, Y., Stroobandt,
D., Van de Peer, Y. <b>A Scalable Hardware Accelerator for Comparing
Protein Sequences</b>. (2006) <i>Proceedings of the First International Conference on
Scalable Information Systems</i>. Hong Kong, April 2006, on CD. <a href="pdf/scalable.pdf">pdf</a></p>

<p>Bonnet, E., Wuyts, J., Rouz&eacute;, P., Van de Peer, Y. <b>Evidence that
microRNA precursors, unlike other non-coding RNAs, have lower folding free
energies than random sequences</b>. (2004) <i>Bioinformatics</i> 20, 2911-7. <a href="pdf/5.pdf">pdf</a>
<a href="http://bioinformatics.oxfordjournals.org/content/20/17/2911.abstract" target="_blank">web</a>
</p>

<p>Bonnet, E.*, Wuyts, J.*, Rouz&eacute;, P., Van de Peer, Y. <b>Detection of 91
potential conserved plant microRNAs in Arabidopsis thaliana and Oryza sativa
identifies important new target genes</b>. (2004) <i>Proc. Natl. Acad. Sci. USA</i> 101, 11511-6. <a href="pdf/4.pdf">pdf</a>
<a href="http://www.pnas.org/content/101/31/11511.full" target="_blank">web</a>
</p>

<p>Picaud, F., Bonnet, E., Gloaguen, V., Petit, D. <b>Decision Making for
Food Choice by Grasshoppers (Orthoptera: Acrididae): Comparison Between a
Specialist Species on a Shrubby Legume and Three Graminivorous Species</b>.
(2003) <i>Environmental Entomology</i>, 32 (3) 680-688. <a href="pdf/decision_making.pdf">pdf</a></p> 

<p>Petit, D., Picaud, F. and Bonnet, E. <b>Les peuplements d'Orthopt&eacute;ro&iuml;des de la lande sur serpentinite de la Flotte et du Cluzeau (Haute-Vienne).</b> (2003) <i>Annales Scientifiques du Limousin</i>, 14, pp 13-28. <a href="pdf/la_flotte_ASL_2003.pdf">pdf</a> 

<p>Bonnet, E., Van de Peer, Y. <b>zt: a software tool for simple and
partial Mantel tests</b>. (2002) <i>J. Stat. Soft.</i> 7, 1-12. <a href="http://www.jstatsoft.org/v07/i10/" target="_blank">web</a></p>


<p>Bonnet, E. et Petit D. <b>Diversit&eacute;, densit&eacute; et biomasse des Orthopt&egrave;res de la
ZNIEFF des Dauges</b>. (1998) <i>Ann. Sc. du Limousin</i>, num&eacute;ro sp&eacute;cial Dauges, pp
111-120. <a href="pdf/bonnet_dauges_1998.pdf">pdf</a></p>


<p>Bonnet, E., Chabrol, L. et Petit, D. <b>Mise &agrave; jour de l'inventaire des
Orthopt&egrave;res du Limousin</b>. Actes du colloque "Cartographie et Inventaire
des Invert&eacute;br&eacute;s comme contribution &agrave; la gestion des
milleux naturels", s&eacute;rie Patrimoine Naturel, Mus&eacute;um National
d'Histoire Naturelle. <a href="pdf/maj_inventaire_1996.pdf">pdf</a></p>

<p>Bonnet, E., Vilks, A., Lenain, J.F., Petit D. <b>Temporal and structural
analysis of the relationships between orthoptera and vegetation</b>. (1997) <i>Ecologie</i>, 28
(3) 209-216. <a href="pdf/bonnet_97_ecologie.pdf">pdf</a></p>

<p>Bonnet, E., Chabrol, L. et Petit, D. <b>Catalogue pr&eacute;liminaire des Orthopt&egrave;res
du Limousin (Orthoptera)</b>. (1995) <i>Nouv. Revue Ent.</i>, T12, fasc. 1, pp 71-83. <a href="pdf/bonnet_catalogue_1995.pdf">pdf</a></p>

<p>Bonnet, E. et Petit, D. <b>A propos de deux esp&egrave;ces d'Orthopt&egrave;res remarquabkles en Haute-Vienne : <i>Calliptamus italicus</i> et <i>Aiolopus thalassinus</i> </b>. (1994) <i>Annales Scientifiques du Limousin</i>, 10, pp 67-73. <a href="pdf/asl_tome10_1994_article6.pdf">pdf</a></p>



<h2><a name="talks"></a>Talks</h2>

<ul>
<li>[ 2024 ] <b>Single cell data analysis for neurodegenerative diseases.</b> Master Recherche, D&eacute;veloppement et Innovations Th&eacute;rapeutiques Univ. Strasbourg, UE Innovations th&eacute;rapeutiques en Recherche et D&eacute;veloppement. </li>
<li>[ 2024 ] <b>Deep learning et g&eacute;nomique.</b> S&eacute;minaire &eacute;l&egrave;ves Ecole Centrale de Paris.</li>
<li>[ 2023 ] <b>Deep learning, big data and biology.</b> Master Life Sciences and Health seminar (Paris-Saclay University).</li>
<li>[ 2022 ] <b>Analysis of single-cell RNA-seq human PBMC datasets</b> France G&eacute;nomique Annual Meeting.</li>
<li>[ 2019 ] <b>Deep learning and GPUs on the cobalt high-performance cluster</b> France G&eacute;nomique Annual Meeting.</li>
<li>[ 2019 ] <b>Evolution of transcriptomics</b>. <a href="https://norbis.w.uib.no/" target="_blank">NORBIS</a> bioinformatics network seminar. Bergen, Norway.</li>
<li>[ 2019 ] <b> Hi-C and whole-genome bisulfite sequencing techniques and analysis</b>. <a href="https://www.ephe.fr/formations/master/master-biologie-sante-ecologie-bse/epigenetique" target="_blank">Ecole Pratique des Hautes Etudes, seminar for master students</a>, Paris, France.</li>
<li>[ 2018 ] <b> NGS and machine learning projects at the CNRGH</b>. <a href="http://www.irsn.fr/FR/Pages/Home.aspx" target="_blank">Seminar at the Institut de Radioprotection et Suret&eacute; Nucl&eacute;aire</a>, Fontenay-aux-roses, France.</li>
<li>[ 2018 ] <b> ATAC-seq, Hi-C and whole-genome bisulfite sequencing techniques</b>. <a href="https://www.ephe.fr/formations/master/master-biologie-sante-ecologie-bse/epigenetique" target="_blank">Ecole Pratique des Hautes Etudes, seminar for master students</a>, Paris, France.</li>
<li>[ 2016 ] <b> WBS: a computational pipeline for the treatment of whole-genome high-throughput bisulfite sequencing data</b>. <a href="https://ngssymposium.sciencesconf.org/" target="_blank">NGS Symposium 2016</a>, IGBMC, Illkirch, France.</li>
<li>[ 2015 ] <b>NaviCell and ACSN live software demo!</b>. <a href="http://www.jsbi.org/giw-incob2015/index.html" target="_blank">GIW/InCob 2015</a>, Tokyo, Japan.</li>
<li>[ 2015 ] <b>NaviCell Web Service for Data Visualization</b>. <a href="http://user2015.math.aau.dk/" target="_blank">UseR! Conference</a>, Aalborg, Denmark. 
<li>[ 2015 ] <b>Easy navigation and versatile data visualization for biological networks with NaviCell technology
</b>. <a href="https://bivi.co/event/2nd-bivi-annual-meeting" target="_blank">2nd Annual BiVi meeting</a>, Invited Keynote Speaker, The Genome Analysis Centre (TGAC), Norwhich, UK. 
</li>
<li>[ 2013 ] <b>Standards and formats in systems biology</b>. Seminar "Mathematics and Biology of Cancer", 12.06.2013, Institut Curie.</li>
<li>[ 2013 ] <b>BiNoM 2.0, a versatile tool for accessing and analyzing pathways using standard systems biology formats</b>. NUS - IBENS Workshop "Novel genome-wide approaches to decipher transcriptional and epigenetic regulation in mammalian cells. 30 May 2013. ENS, Paris, France.</li>
<li>[ 2012 ] <b>NGS and microRNAs.</b> Workshop "Generation and Analysis of Next Generation Sequence (NGS) data". Invited Keynote Speaker, Gent, Belgium.</li> 
<li>[ 2012 ]  <b>BiNoM, a Cytoscape plugin for accessing and analyzing pathways using standard systems biology formats.</b> Computational Modeling in Biology Network (COMBINE) 2012. August 2012. The Donnelly Centre, University of Toronto, Canada.</li>
<li>[ 2010 ]  <b>Prediction of a prostate cancer regulatory network from gene expression, microRNA and clinical data.</b> 9th European Conference on Computational Biology (ECCB), Ghent, Belgium. </li>
<li>[ 2010 ]  <b>Integrating MicroRNA Expression Data And Other Sources In Module Network Reconstruction.</b> BIOMS workshop Heidelberg. </li>
<li>[ 2010 ]  <b>Top down systems biology: application to the inference of a module network from cancer data including microRNAs.</b> Institut Curie, Paris.</li> 
<li>[ 2010 ]  <b>Module network inference from a cancer gene expression data set identifies microRNA regulated modules.</b> VIB Science Club Genetics IV, Leuven, Belgium.</li> 
<li>[ 2009 ]  <b>MicroRNA module network inference from expression data.</b> Institut Pasteur, Paris.</li> 
<li>[ 2008 ]  <b>MicroRNA module network inference from expression data.</b> 4th EMBO Conference "From Functional Genomics to Systems Biology", Heidelberg, Germany.</li>  
<li>[ 2006 ]  <b>MicroARNs, comparaison et pr&eacute;diction.</b> Ecole chercheurs phylog&eacute;nomique, Carry-le-Rouet.</li> 
<li>[ 2005 ]  <b>Computational screening for Arabidopsis, rice and poplar microRNAs: have most conserved plant miRNAs been identified?</b> Plant GEMS meeting, Amsterdam.</li> 
<li>[ 2004 ]  <b>Computational identification of Arabidopsis microRNAs. </b>Annual VIB seminar, Blankenberge, Belgium.</li> 
</ul>


<h2><a name="training"></a>Training</h2>

<p>
<ul>
<li><b>[ 2024 ]</b> Manager les singularit&eacute;s. </li>
<li><b>[ 2023 ]</b> R&eacute;ussir ses entretiens annuels. </li>
<li><b>[ 2023 ]</b> Risques psychosociaux &eacute;l&eacute;ments clef de la pr&eacute;vention. </li>
<li><b>[ 2023 ]</b> Secouriste sauveteur au travail (SST). </li>
<li><b>[ 2022 ]</b> <a href="https://www.fun-mooc.fr/en/courses/advances-stem-cell-biology/" target="_blank">FUN MOOC Advances in Stem Cell Biology</a>. </li>
<li><b>[ 2021 ]</b> <a href="https://www.fun-mooc.fr/en/courses/epigenetics/" target="_blank">FUN MOOC Epigenetics</a>. </li>
<li><b>[ 2021 ]</b> Formation Salari&eacute; Secouriste du Travail (SST). </li>
<li><b>[ 2019 ]</b> <a href="https://courses.nvidia.com/courses/course-v1:DLI+C-AC-01+V1/about" target="_blank">Fundamentals of Accelerated Computing with CUDA C/C++</a>. Online Nvidia course, 8 hours.</li>
<li><b>[ 2019 ]</b> <a href="https://courses.nvidia.com/courses/course-v1:DLI+C-MG-01+V1/about" target="_blank">Fundamentals of Deep Learning for multi-GPUs</a>. 1 day Nvidia training at the CEA computing centre CCRT.</li>
<li><b>[ 2019 ]</b> <a href="https://indico.in2p3.fr/event/18772/" target="_blank">Workshop GPU CC-IN2P3</a>. CNRS IN2P3 data center Lyon, training 3 days.</li>
<li><b>[ 2018 ]</b> Single cell transcriptome tutorial. 4 hours, ISMB 2018.</li> 
<li><b>[ 2018 ]</b> Deep learning for network biology tutorial. 4 hours, ISMB 2018.</li> 
<li><b>[ 2018 ]</b> Proximity management. CEA internal training 24 hours.</li> 
<li><b>[ 2018 ]</b> Management and law. CEA internal training 20 hours.</li> 
<li><b>[ 2016 ]</b> <a
href="https://www.udacity.com/course/deep-learning--ud730" target="_blank">Deep
Learning by Google - Udacity</a>. 2 months. 
</li>

<li><b>[ 2015 ]</b> MOOC Coursera.org <a
href="https://www.coursera.org/course/datascitoolbox" target="_blank">The Data
Scientist&#39;s Toolbox</a>. 4 weeks. Johns Hopkins University. </li> 

<li><b>[ 2015 ]</b> MOOC Coursera.org <a
href="https://www.coursera.org/course/predmachlearn" target="_blank">Practical
Machine Learning</a>. 4 weeks. John Hopkins University. </li> 

<li><b>[ 2014 ]</b> MOOC Coursera.org <a
href="https://www.coursera.org/course/datasci" target="_blank">Introduction to
data science</a>. 8 weeks. University of Washington. </li> 
</ul>
</p>


<h2><a id="soft"></a>Software and Models</h2>

<p><ul>
<li> Using convolutional neural networks for the classification of breast cancer images <a href="https://github.com/erbon7/pcam_analysis" target="_blank">GitHub repository</a></li>
<li> ACSNMineR <a href="https://github.com/sysbio-curie/ACSNMineR" target="_blank">GitHub repository</a></li>
<li> RNaviCell, a R package for visualizing large-scale "omics" data on ACSN molecular maps. <a href="https://github.com/sysbio-curie/RNaviCell" target="_blank">GitHub repository</a></li>
<li> JNaviCell, a Java package for visualizing large-scale "omics" data on ACSN molecular maps <a href="https://github.com/sysbio-curie/JNaviCell" target="_blank">GitHub repository</a></li>
<li> Prostate Cancer DREAM Challenge 2015. <a href="https://www.synapse.org/#!Synapse:syn4485029/wiki/234262" target="_blank">model repository (synapse)</a></li>

<li>BiNoM, a a toolbox for accessing and analyzing biological pathways. <a href="http://binom.curie.fr" target="_blank">homepage</a></li>
<li>Lemon-Tree is a Java package for the inference of module networks from high-throuput data sets. <a href="https://github.com/eb00/lemon-tree" target="_blank">GitHub repository</a></li>

<li> DREAM Breast Cancer Prognosis Challenge 2013. <a href="http://sagebase.net/WP/blog/2013/01/10/sage-bionetworks-dream-breast-cancer-prognosis-challenge/">Dream Challenge Website</a></li>
<li>Tapir, a web portal for the prediction of plant microRNA targets. <a href="https://github.com/eb00/tapir" target="_blank">GitHub repository</a></li>
<li>MirFinder, a computational pipeline designed for the <i>de novo</i> prediction of plant microRNAs. <a href="http://www.pnas.org/content/101/31/11511.full" target="_blank">publication</a></li>
<li>Randfold, a software for permutation test of microRNA precursor sequences. <a href="https://github.com/eb00/randfold" target="_blank">GitHub repository</a></li>
<li>zt is a software for testing correlation in distance matrices, using a permutation test. <a href="https://github.com/eb00/zt" target="_blank">GitHub repository</a></li>
</ul>
</p>

<h2><a id="grants"></a>Grants and Projects</h2>
<p>
<ul>

<li>2023 Association France Alzheimer grant for the scientific project "Uncovering the role of astrocytes in Alzheimer pathology in a non-human primate". Project PI, collaboration with Dr Marc Dhenain (CEA MIRCen), Dr Carole Escartin (CEA / MIRCen) and Dr H. Hirbec (IGF Montpellier).</li> 
<li>2023 Paris-Saclay Life Sciences and Health grant for the scientific project "Age and Alzheimer&apos;s disease-related transcriptomic changes in astrocytes". Project PI, collaboration with  with Dr Marc Dhenain (CNRS / MIRCEN, CEA).</li> 
<li>2021 Association France Alzheimer grant for the scientific project "Scientific therapeutic targeting of reactive astrocytes subpopulations in Alzheimer disease". Project co-writer with Dr Carole Escartin (CEA/MIRCen, PI).</li>
<li>2019 CEA grant for PhD project "Glial cells and elimination of toxic aggregates in Huntington disease". Project co-writer with Dr Carole Escartin (CNRS / MIRCEN, CEA).</li> 
<li>2019 Fondation Maladie Rares - Transcriptomics for the study of the role of Astrocytes and Glial Cells in Huntington's disease. Co-writer with Dr Carole Escartin (CNRS / MIRCEN, CEA).</li>
<li>2010 - 2011 <a href="https://cordis.europa.eu/project/rcn/88080/factsheet/en" target="_blank">Apo-Sys</a>, Apoptosis systems biology applied to cancer and AIDS (workpackage leader).</li>
<li>2010 Experimental validation of gene targets for an inferred module network in Arabisopsis. VIB special fund for novel experimental techniques (project leader).</li>
<li>2008 - 2010 MoneyMaize, funded by IWT, identification of molecular networks for yield increase in maize (project co-writer, workpackage leader).</li>
<li>2007 - 2009 <a href="http://www.bioframe.net/summary.html" target="_blank">Bioframe</a>, funded by IWT, conception of a bioinformatics platform for integrative modeling in systems biology (project co-writer, workpackage leader).</li>
<li>2007 - 2011 <a href="http://flexware.elis.ugent.be" target="_blank">Flexware</a>, funded by IWT, exploitation of flexible hardware platforms for massively parallel bioinformatics application (project co-writer, workpackage leader).</li>
<li>2005 IWT funding for an hardware accelerated BLAST computer system.</li>
<li>2002 - 2005 <a href="https://cordis.europa.eu/project/rcn/60159/factsheet/en" target="_blank">Planet</a>, a network of european plant databases FP5 project (ERC funding), workpackage leader.</li>
</ul>
</p>

<h2>Referee experience</h2>
<p>
<ul>
<li>Review college panel expert for the Belgian FWO (2021 - 2023).</li>
<li>Grant reviewer for the Canadian FCI, french ANR, belgian IWT, belgian FWO, european ERC, US NSF, US USDA.</li>
<li>Manuscript referee for Nucleic Acids Research, Bioinformatics, RNA, Trends in Genetics, PLOS Computational Biology, PLOS One, Scientific Reports, Scientific Data, Genome Biology, BMC Bioinformatics, BMC Plant Biology, Plant Physiology, Bioessays, The Plant Journal, BMC genomics, Plant Biology, Journal of experimental botany, Tree Genetics and Genomes, Molecular Genetics and Genomics, FEBS Journal.</li>
</ul>
</p>


<h2>Teaching experience</h2>
<p><ul>
<li>2019 Ecole Pratique des Hautes Etudes (EPHE) in Paris; Master Biology, Health and Ecology; Hi-C and whole-genome bisulfite analysis; 2h (in french).</li> 
<li>2019 CNRGH; Internal training for biologists; R package methylKit for the analysis of methylome data; 2h (in french).</li> 
<li>2018 Ecole Pratique des Hautes Etudes (EPHE) in Paris; Master Biology, Health and Ecology; ATAC-seq, Hi-C and whole-genome bisulfite analysis; 2h (in french).</li> 
<li>2008 - 2010	Ghent University; Master Biochemistry and Bioinformatics; Unix for bioinformatics; 30h (in english).</li>
<li>2008 - 2010	Ghent University; Master Biochemistry and Bioinformatics; Relational database systems; 30h (in english).</li>
<li>1994 - 1996	University of Limoges; DEUG Biology (undergraduate level); 15 h population biology practical sessions (in french).</li>
</ul>
</p>

<h2>Supervision</h2>
<p><ul>
<li>2023-2026 Altynbek Zhubanchaliyev, PhD student in bioinformatics (co-supervision with L. Martignetti, Institut Curie).</li>
<li>2023 Clara Cherruaud, M1 student &amp; ISEN Brest engineering school. </li>
<li>2022 Jeanne Cherruaud, MD student &amp; Master student Biologie et Sant&eacute;. </li>
<li>2019-2023 Miriam Riquelme-Perez, PhD student in bioinformatics (co-supervision with C. Escartin and S. Brohard).</li>
<li>2018 Maria Kondili, Bioinformatics engineer, (co-supervision with S. Chantalat).</li> 
<li>2018 Sarah Benmohammed, Bioinformatics engineer, (co-supervision with S. Chantalat).</li>
<li>2016 - 2018 Yimin Shen, Bioinformatics engineer, (co-supervision with J. Tost).</li>
<li>2014 Nadir Sella, Bioinformatics engineer, U900 Bioinformatics and Computational Biology of Cancer, Institut Curie (co-supervision with Andrei Zinovyev).</li> 
<li>2010 Ying He, Bioinformatics engineer, Bioinformatics and Evolutionary research group, Ghent.</li>
<li>2009 Irina Lambertz, Post-doctoral researcher, VIB Department for Molecular Biomedical Research (co-supervision with Jean-Christophe Marine).</li> 
<li>2006 J.M. Sanchez, Ghent University, Master 2 student (co-supervision with Dirk Stroobandt and Yvan Saeys).</li>
<li>2005 Sven De Smet, Ghent University, Master 2 student (co-supervision with Dirk Stroobandt and Yvan Saeys).</li>
<li>2005 Bram Minnaert, Ghent University, Master 2 student (co-supervision with Dirk Stroobandt and Yvan Saeys).</li>
</ul>
</p>

<h2>Workshops organisation</h2>
<p><ul>
<li>2016 Co-organiser with Dr A. Joshi and Dr T. Michoel, workshop Network Inference: new methods and new data, 1 day, European Conference on Computational Biology, The Hague, The Netherlands.</li> 
<li>2013 Co-organiser of the workshop COMBINE 2013 (Computational Modeling in Biology Network), Institut Curie, Paris, 5 days, more than 100 participants.</li>
</ul>
</p>

<h2>PhD and HDR committees</h2>
<p><ul>
<li>2024 HDR, examinateur and jury member, Gautier Stoll, Paris-Cit&eacute; University.</li>
<li>2024 examinateur, PhD Thesis Jorge Bretones Santamarina, "Integrated multiomic analysis, synthetic lethality inference and network pharmacology to identify SWI/SNF subunit-specific pathway alterations and targetable vulnerabilities", Institut Curie, Paris-Saclay University.</li>
<li>2023-2026 PhD monitoring Committee, Jennifer Thamara Mata Orozco, "Therapeutic antibody immunogenicity: large-scale single-cell analysis of adalimumab-specific CD4+ T cells", CEA and Paris-Saclay University.</li>
<li>2024-2027 PhD monitoring Committee, Cl&eacute;mence Raimond, "Etude de la d&eacute;g&eacute;n&eacute;rescence des cellules oligodendrocytaires et &eacute;tude des effets de la stimulation magn&eacute;tique r&eacute;p&eacute;titive transpinale dans le d&eacute;veloppement de la SLA chez la souris", SPPIN and Paris-Cit&eacute; University.</li>
<li>2023-2026 PhD monitoring Committee, Li&eacute;baut Dudragne, "Genomic Instability in S. Cerevisiae", Sorbonne University.</li>
<li>2023-2026 PhD monitoring Committee, Liangwei Yin, "Bioinformatics and Artificial Intelligence Methods to Assess Organoids and Tumor Spheroids Models", Grenoble-Alpes University.</li>
<li>2022 rapporteur, PhD thesis Matthieu Cornet, "Etude informatique des effets cliniques et omiques des modulateurs de CFTR dans la mucoviscidose et recherche de nouvelles cibles", Ecoles des Mines de Paris &amp; PSL University, Paris, France.</li>
<li>2021 examinateur, PhD thesis Yunfeng Wang, "Transcriptome and Genome Analysis using Mapping-free Protocols", Paris-Saclay University, Paris, France.</li>
<li>2018 examinateur, PhD thesis S. Grison, "Approches omics pour les faibles doses sur un mod&egrave;le rat multig&eacute;n&eacute;rationnel", IRSN, Paris, France.</li>
<li>2016 examinateur de mi-th&egrave;se, A. Trolet, "R&eacute;gulation et fonction de la prot&eacute;ine AGO1 au cours du cycle cellulaire dans les cellules BY2 de tabac et chez Arabidopsis thaliana", Institut de Biologie Mol&eacute;culaire des Plantes CNRS, Strasbourg, France.</li>
<li>2010 examinateur, PhD thesis N. Naouar, "Methods for genome-wide expression analysis", VIB department of Plant Systems Biology, Gent, Belgium.</li>
</ul>
</p>


<p>
<script type="text/javascript" id="clustrmaps" src="//clustrmaps.com/map_v2.js?d=a3UVVyuB3ix7MA_bZS5sDMcfmLCweCWREItRcMwNPQw&cl=ffffff&w=a"></script>
</p>

</body>
</html>
